src/dos/dos_mscdex.cpp

```diff
@@ -308,7 +308,8 @@ int CMscdex::AddDrive(Bit16u _drive, char* physicalPath, Bit8u& subUnit)
 		break;
 	case MountType::DIRECTORY:
 		LOG(LOG_MISC,LOG_NORMAL)("MSCDEX: Mounting directory as cdrom: %s", physicalPath);
-		LOG(LOG_MISC,LOG_NORMAL)("MSCDEX: You wont have full MSCDEX support!");
+		LOG(LOG_MISC, LOG_NORMAL)
+		("MSCDEX: You won't have full MSCDEX support!");
```

Comment:
@dreamer (clang got confused here, which you've mentioned before; can you manually fix it on merge?)